src/index.html

```diff
@@ -12,7 +12,50 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="stars__body">
+    <div class="wrapper">
+      <div class="stars stars--0">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--2">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--3">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--4">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--5">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+    </div>
```

Comment:
Consider using semantic tags to improve the structure and readability of your HTML content. Instead of multiple <div> elements, you might use <section> or <article> tags to group related items together semantically.